examples/persistence_transform.py

```diff
@@ -0,0 +1,45 @@
+"""
+========================================
+Persistence Transform in Mapcubes
+========================================
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+"""
+import matplotlib.pyplot as plt
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+import copy
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+###############################################################################
+# For a data set consisting of N images with intensity values I(x,y,t), the Persistence Map Pn is a
+# function of several arguments, namely intensity, location and time:
+# Pn(x,y,tn)=Q(I(x,y,t≤tn))
+maps_data = []
+persistence_maps = []
+persistence_maps.append(aiamapcube[0])
+maps_data.append(aiamapcube[0].data)
+for i in range(1, len(aiamapcube)):
+    value = aiamapcube[i].data/aiamapcube[i].data.max() < maps_data[i-1]
```

Comment:
Are you sure you want to normalise the values you compare and then add to the persistence map? Here `value` will be true for most of the image because you are comparing:
[min/max(), 1] with [min, max]

So I would think the correct operation would be:
```python
value = aiamapcube[i].data < maps_data[i-1]
```
or
```python
value = aiamapcube[i].data/aiamapcube[i].data.max() < maps_data[i-1]/maps_data[i-1].max()
```

Reply:
ok, I get it. I was doing the wrong way